08_project/mohankrishna/pkg/mohankrishna-puppy/store/sync_store.go

```diff
@@ -0,0 +1,55 @@
+package store
+
+import (
+	"fmt"
+	"sync"
+
+	types "github.com/anz-bank/go-training/08_project/mohankrishna/pkg/mohankrishna-puppy"
+)
+
+type SyncStore struct {
+	sm sync.Map
```

Comment:
The sync store should be safe to use by multiple concurrent users. You should add a `mutex` to this class and wrap the necessary function with `mutex.Lock()` and `mutex.Unlock()`. See the relevant comments.